### PR TITLE
Update GeometryOperators.py

### PR DESCRIPTION
**Pull request by PipKat**
@maxcapodi78 Lines 272 through 310 are almost identical--except for the return value. Is this correct?

This is a large file with lots of math equations. I removed the use of variables in the function description if there  wasn't a subsequent formula that used  it. My goal was to make it more readable/understandable.

### CODE REVIEW

File: pyaedt/modeler/GeometryOperators.py [hunk 1]
```diff
@@ -137,6 +138,8 @@ def cs_plane_str(val):
 
         Returns
         -------
+        str
+           String for the coodinate system plane.
```
Comment (PipKat) [inline pyaedt/modeler/GeometryOperators.py:142]:
```suggestion
           String for the coordinate system plane.
```

File: pyaedt/modeler/GeometryOperators.py [hunk 2]
```diff
@@ -265,83 +271,84 @@ def v_cross(a, b):
...
         """
         return GeometryOperators._v_dot(a, b)
 
     @staticmethod
     @aedt_exception_handler
     def v_prod(s, v):
-        """ Evaluates the product between the scalar s and the vector v.
-        Vector v can be any length.
-
+        """Evaluate the product between a scalar v alue and a vector.
```
Comment (PipKat) [inline pyaedt/modeler/GeometryOperators.py:315]:
```suggestion
        """Evaluate the product between a scalar value and a vector.
```

File: pyaedt/modeler/GeometryOperators.py [hunk 3]
```diff
@@ -454,18 +462,19 @@ def points_distance(p1, p2):
...
+        """Find a point on a plane.
 
         Parameters
         ----------
         pointlists : list
-            
-        direction : int
-             The default is `0`
+            List of points.          
+        direction : int, optinal
```
Comment (PipKat) [inline pyaedt/modeler/GeometryOperators.py:471]:
```suggestion
        direction : int, optional
```

File: pyaedt/modeler/GeometryOperators.py [hunk 4]
```diff
@@ -870,20 +874,21 @@ def quaternion_to_axis_angle(q):
...
 
         theta : float
-            angle of rotation in radians
+            Angle of rotation in radians.
 
         Returns
         -------
         list
-            quaternion [q1, q2, q3, q4]
+            List of ``[q1, q2, q3, q4]`` ccordinates for the quaternion.
```
Comment (PipKat) [inline pyaedt/modeler/GeometryOperators.py:890]:
```suggestion
            List of ``[q1, q2, q3, q4]`` coordinates for the quaternion.
```

File: pyaedt/modeler/GeometryOperators.py [hunk 5]
```diff
@@ -265,83 +271,84 @@ def v_cross(a, b):
...
+        """Evaluate the dot product between two geometry vectors.
 
         Parameters
         ----------
         a : list
-            first vector [x, y, z]
-
+            List of ``[x, y, z]`` coordinates for the first vector. 
         b : list
-            second vector [x, y, z]
```
Comment (maxcapodi78) [inline pyaedt/modeler/GeometryOperators.py:276]: you can delete docstrings for private members (ones with _ as prefix in name). those methods will be ignored into documentation generation

**Review by maxcapodi78 — APPROVED** [review]

**Pull request merged**